### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by kangzongxian**
Improve grammar and styling
Ensure correctness
Updated tables

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -545,10 +544,10 @@ Listed here are some frequently asked questions that we hope will be helpful for
...
 
 **Q**: Why am I getting error messages for fields that are correct, or that I'm not editing?<br>
-**A**: If you are using the `add`/`edit` commands and input the `name`/`request` fields, ensure that they do not contain any **field prefixes** after whitespace in their content (e.g., `rq/Extra bed & p/c`), as GuestBook could be parsing part of your input as another field. Consider using capital letters as this check is case-sensitive (`rq/Extra bed & P/C` will be parsed correctly). You may refer to [Field summary](#field-summary) for more information about field constraints.
+**A**: If you are using the `add` or `edit` commands and inputting the `name` or `request` fields, ensure that they **do not contain any field prefixes** after whitespace in their content (e.g., `rq/Extra bed & p/c`), as GuestBook could be parsing part of your input as another field. **Consider using capital letters** as this check is case-sensitive (e.g, `rq/Extra bed & P/C` will be parsed correctly as part of a `request`). You may refer to [Field summary](#field-summary) for more information about field constraints.
```
Comment (bryanhce) [inline docs/UserGuide.md]: Should this be "e.g.," ?

Comment (bryanhce) [reply docs/UserGuide.md]: Maybe we can avoid using "parse" here as it seems technical for our target audience. Perhaps "(e.g., `rq/Extra bed &p/c`), as GuestBook may mistake part of your input as another field." Likewise in the subsequent part of this line "... will be handled correctly as part of a `request`)" ?

**Review by bryanhce — CHANGES_REQUESTED** [review]
Nice work! Small change

**Review by bryanhce — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (kangzongxian) [reply docs/UserGuide.md]: Agreed!

**Pull request merged**